packages/inscription-view/src/components/editors/part/Part.test.tsx

```diff
@@ -56,69 +61,70 @@ describe('Part', () => {
   });
 
   test('render', () => {
-    renderPart(generalPart);
-    assertExpanded('General', false);
+    renderPart([generalPart]);
+    assertActive('General', 'active');
   });
 
   test('state', () => {
-    renderPart(generalPart);
+    renderPart([generalPart]);
     assertPartState('General', undefined);
-    renderPart(callPart);
+    renderPart([callPart]);
     assertPartState('Call', 'warning');
-    renderPart(resultPart);
+    renderPart([resultPart]);
     assertPartState('Result', 'error');
   });
 
   test('reset data', async () => {
     let dirty = true;
     const action = () => (dirty = false);
-    renderPart({ ...callPart, reset: { dirty, action } });
-    await userEvent.click(screen.getByRole('button', { name: 'Reset Call' }));
+    renderPart([{ ...generalPart, reset: { dirty, action } }]);
+    await userEvent.click(screen.getByRole('button', { name: 'Reset General' }));
     expect(dirty).toBeFalsy();
   });
 
-  test('open section', async () => {
-    renderPart(generalPart);
-    const trigger = screen.getByRole('button', { name: 'General' });
-    assertExpanded('General', false);
+  test('open tab', async () => {
+    renderPart([generalPart, callPart]);
+    assertActive('General', 'active');
+    assertActive('Call', 'inactive');
+    const trigger = screen.getByRole('tab', { name: 'Call' });
     await userEvent.click(trigger);
-    assertExpanded('General', true);
-    await userEvent.click(trigger);
-    assertExpanded('General', false);
+    assertActive('General', 'inactive');
+    assertActive('Call', 'active');
   });
 
   test('open section by keyboard', async () => {
-    renderPart(callPart);
-
-    const trigger = screen.getByRole('button', { name: 'Call' });
+    renderPart([generalPart, callPart]);
 
-    await userEvent.tab();
+    const trigger = screen.getByRole('tab', { name: 'General' });
+    trigger.focus();
```

Comment:
I think this is wrong. you should use userEvent for such things. maybe also the reason why you needed to add an wait for?
why not just click on the first tab and then do right left stuff?